antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -644,6 +658,7 @@ def create_variant_study(self, uuid: str, name: str, params: RequestParameters)
             additional_data=additional_data,
         )
         self.repository.save(variant_study)
+        self._copy_parent_ts_generation_info(study.id, new_id)
```

Comment:
as discussed, should go at the start of snapshot generation, it's part of the same "copy parent" operation that we do at the beginning of generation